pod.go

```diff
@@ -718,6 +718,10 @@ func (p *Pod) start() error {
 		return err
 	}
 
+	for _, c := range p.containers {
+		c.storeMounts()
```

Comment:
I think you should remove that and store the mount points from the agent itself.